apps/api/src/request/authorization/permissions.js

```diff
@@ -1,5 +1,8 @@
-const ownProductionManagement = (user, from, to) => {
-    return user.productionManagementIds.includes(from.productionManagementId);
+const ownProductionManagement = (user, from) => {
+    return (
+        user.role === "admin" ||
```

Comment:
Much better than setting all production managements to the admin profile every time 👍